kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/steps/KafkaSteps.java

```diff
@@ -52,7 +52,7 @@ public static void createTopic(String deployNamespace, String topicName, String
         Job adminClientJob = TestClientsJobTemplates.defaultAdminClientJob(name, args).build();
         kubeClient().getClient().batch().v1().jobs().inNamespace(deployNamespace).resource(adminClientJob).create();
         String podName = KafkaUtils.getPodNameByLabel(deployNamespace, "app", name, Duration.ofSeconds(30));
-        DeploymentUtils.waitForPodRunSucceeded(deployNamespace, podName, Duration.ofSeconds(30));
+        DeploymentUtils.waitForPodRunSucceeded(deployNamespace, podName, Duration.ofMinutes(1));
```

Comment:
I'm slightly surprised that creating/deleting can take 30seconds+.  What's causing the delay?
We really ought to extract the value as a environment too to allow tuning to be performed for a particular execution environment.

Reply:
I don't know what is the root cause, but as working with RHOSAK working with OCP/OSD could take longer than expected sometimes... We have many headaches with that in the past.

Reply:
Yeah I'd echo @k-wall's thoughts we should externalise the timeouts and try and understand why it takes so long.